ui/src/app/utils/access-utils.tsx

```diff
@@ -195,8 +191,6 @@ export const getAccessModuleConfig = (
         ),
         adminPageTitle: 'Verify your identity with Login.gov',
         refreshAction: () => redirectToRas(false),
-        isRequiredByRT: enforceRasLoginGovLinking,
```

Comment:
I don't understand deleted code here. So we don't need "calculate" `isRequiredByRT` by `enforceRasLoginGovLinking` anymore because we're getting values directly from DB table, right? How does DB table `access_module` knows how to update `requiredForRTAccess` and `requiredForCTAccess` by `enforceRasLoginGovLinking`?

Reply:
Nice catch!

We do still need to do this, for a little while.  I'll put this back and add a comment.